main/reference/zoe-api/zoe.md

```diff
@@ -286,73 +286,63 @@ const { creatorFacet, publicFacet, creatorInvitation } = await E(Zoe).startInsta
 ```
 
 
-## E(Zoe).offer(invitation, proposal?, paymentKeywordRecord?, offerArgs)
-- **invitation**: **[Invitation](./zoe-data-types.md#invitation) | Promise&lt;Invitation>**
-- **proposal**: **Proposal** - Optional.
-- **paymentKeywordRecord**: **PaymentKeywordRecord** - Optional.
-- **offerArgs**: **Object**
+## E(Zoe).offer(invitation, proposal?, paymentKeywordRecord?, offerArgs?)
+- **invitation**: **[Invitation](./zoe-data-types.md#invitation) | Promise&lt;[Invitation](./zoe-data-types.md#invitation)>**
+- **proposal**: **[Proposal](/glossary/#proposal)** - Optional.
+- **paymentKeywordRecord**: **[PaymentKeywordRecord](./zoe-data-types.md#keywordrecord)** - Optional.
+- **offerArgs**: **[CopyRecord](/glossary/#passable)** - Optional.
```

Comment:
comment: not having these kept in sync with agoric-sdk by machine is just maddening.

 - https://github.com/Agoric/agoric-sdk/issues/4291

Reply:
> comment: not having these kept in sync with agoric-sdk by machine is just maddening

Indeed!